### ISSUE TITLE
Include sidebar summary in page.

### ISSUE THREAD

**Issue by kfogel** [OpenTechStrategies/torque-sites#44]
As discussed with Jeff.  This can probably all be done via `sec_map`.  Maybe some minor enhancements to csv2wiki will be needed to enable the summary to be distinctively formatted; we'll see.

These are the summary fields for this round:

>Review Number                                  
>Organization Legal Name                        
>Country                                        
>Locality                                       
>Primary Contact First Name                     
>Primary Contact Last Name                      
>Primary Contact Title                          
>Primary Contact Email                          
>Organization Legal Status                      
>Project Title                                  
>Project Description                            
>Executive Summary                              
>Youtube Video Link                             
>Priority Populations                           
>Primary Subject Area                           
>Location Of Current Solution Country           
>Location Of Current Solution2 Country          
>Location Of Current Solution3 Country          
>Location Of Future Work Country                
>Sustainable Development Goals                  
>Key Words                                      
>Annual Operating Budget                        
>Number of Employees

**Comment by kfogel** [comment]
https://www.mediawiki.org/wiki/Manual:Table_of_contents tells how to control presence, depth, and positioning of the Table of Contents that MediaWiki generates by default.

**Comment by kfogel** [comment]
Okay, new understanding: the summary is a sidebar, like the one on https://en.wikipedia.org/wiki/Battle_of_Thermopylae.  (Pages already have a TOC -- I just forgot that, but looking at one now confirms it.  We might change the default depth of that TOC, but that's easy and is a separate issue anyway.)

The exact content of that sidebar is something we're still working out, but it'll be more or less the fields listed earlier, though we'll have to see if Project Description and Executive Summary are too long to fit well -- if they are, we could do some data preprocessing to create a new "Truncated Summary" column that gets used only in the sidebar summary, or something like that.

**Comment by frankduncan** [comment]
This is largely copied from my comments in Zulip, but I figured this is the more appropriate place for it.

Attached is a potential implementation based on the initial work you did.  I remember you didn't want the video embedded there, and I just wanted to see what it looked like. I do think it looks good, but I wanted to put it out there as an example

![example_wiki](https://user-images.githubusercontent.com/5125316/63686066-93e19f80-c7c6-11e9-9d66-171982b2c4ba.png)

However, I also did a bit more research on how wikipedia does the sidebar, and it looks like they have a template for these kinds of things, and that's how normally things are done. You do the styling all in the template, and then in your wiki page, you just do {{TemplateName... }} with a whole lot of values. That would make the secmap a bit prettier without a bunch of html in it, and it allows us to be a little more expressive with the body of the sidebar in terms of css, but we would need to ensure we deploy the template as part of csv2wiki or the ansible work, as well as learn all the stuff that goes into templating (not hard, but not turnkey).

I did take a shot at creating templates, and it was not extremely straightforward to take arbitrary text and turn it into sidebar. Templates play much better with key/value pairs, so we'd want to either use the key/value pairs as mentioned above, with their own styling, or have some kind of catch all.
